openslides/motions/static/js/motions/site.js

```diff
@@ -1065,6 +1078,49 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions'])
     }
 ])
 
+.controller('CategorySortCtrl', [
+    '$scope',
+    '$stateParams',
+    '$http',
+    'MotionTree',
+    'Category',
+    'category',
+    'Motion',
+    'motions',
+    function($scope, $stateParams, $http, MotionTree, Category, category, Motion, motions) {
+        Category.bindOne(category.id, $scope, 'category');
+        Motion.bindAll({}, $scope, 'motions');
+        $scope.filter = { category_id: category.id,
+                          orderBy: 'identifier' };
+
+        $scope.$watch(function () {
+              return Motion.lastModified();
+        }, function () {
+            $scope.items = MotionTree.getTree(Motion.filter($scope.filter));
+        });
+
+        $scope.alert = {};
+        // Numbers all motions in this category by the given order in $scope.items
+        $scope.numbering = function (category) {
```

Comment:
The `category` is part of the scope so we do not have to give it as function argument.